test/components/connect.spec.tsx

```diff
@@ -2241,40 +2665,39 @@ describe('React', () => {
     })
 
     describe('Refs', () => {
-      it.skip('should throw when trying to access the wrapped instance if withRef is not specified', () => {
-        const store = createStore(() => ({}))
-
-        class Container extends Component {
-          render() {
-            return <Passthrough />
-          }
-        }
-
-        const decorator = connect((state) => state)
-        const Decorated = decorator(Container)
-
-        class Wrapper extends Component {
-          render() {
-            return (
-              <Decorated ref={(comp) => comp && comp.getWrappedInstance()} />
-            )
-          }
-        }
-
-        // TODO Remove this when React is fixed, per https://github.com/facebook/react/issues/11098
-        const spy = jest.spyOn(console, 'error').mockImplementation(() => {})
-        expect(() =>
-          rtl.render(
-            <ProviderMock store={store}>
-              <Wrapper />
-            </ProviderMock>
-          )
-        ).toThrow(
-          `To access the wrapped instance, you need to specify { withRef: true } in the options argument of the connect() call`
-        )
-        spy.mockRestore()
-      })
-
+      //   it.skip('should throw when trying to access the wrapped instance if withRef is not specified', () => {
```

Comment:
:hand: Yeah, this is dead - `getWrappedInstance()` went away in version 6, and I guess we never removed this test.  Delete it.

Reply:
![image](https://user-images.githubusercontent.com/53305259/127748417-2ba537ab-5ce0-42a2-847a-80a11185b49f.png)
This descript is also all skip, I can also delete it directly?

Reply:
yeah, let's delete that one too